examples/src/main/scala/sqlest/examples/Extractor.scala

```diff
@@ -21,6 +21,25 @@ object ExtractorExamples extends App with DatabaseExample {
     name = FruitTable.name,
     juiciness = FruitTable.juiciness
   )
+
+  // extractors can be used in the declaration of other extractors
+  lazy val smoothyExtractor = extract[Smoothy](
+    description = SmoothyTable.description,
+    fruits = fruitExtractor.asList
+  ).groupBy(SmoothyTable.id)
+
+  // inner joins can be used as follows
+  val smoothies =
+    select(smoothyExtractor.columns)  // Extractors maintain a list of columns they use
```

Comment:
It's now not necessary to specify `smoothyExtractor.columns`. When extract... is called it sets the fields to select on the select statement.

This line can just be replaced with `select`